README.rst

```diff
@@ -0,0 +1,113 @@
+FIXME - testing correctness of these links
+.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
+   :target: https://travis-ci.org/diffpy/diffpy.pdfmorph
+
+.. image:: http://codecov.io/github/diffpy/diffpy.pdfmorph/coverage.svg?branch=master
+   :target: http://codecov.io/github/diffpy/diffpy.pdfmorph?branch=master
+
+
+PDFmorph
+========================================================================
+# FIXME
+add description, sample below
+Code package for morphing atomic pair distribution functions.
+
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, 
+https://github.com/diffpy/diffpy.utils
+
+# FIXME - add link for instructions
+If you can't use conda and need to build from the sources, instructions are [here].
+
+INSTALLATION
+------------------------------------------------------------------------
+
+The preferred method is to use Anaconda Python and install from the
+"diffpy" channel of Anaconda packages ::
+
+     conda config --add channels diffpy
+     conda install diffpy.pdfmorph
+
+# FIXME
+PDFmorph can then be started from a terminal...
+[insert more instructions here]
+
+With Anaconda, PDFmorph can be later upgraded to the latest released
+version using ::
+
+     conda update diffpy.pdfmorph
+
+With other Python distributions the program can be upgraded to
+the latest version as follows ::
+
+     easy_install --upgrade diffpy.pdfmorph
+
+
+DEVELOPMENT
+------------------------------------------------------------------------
+
+PDFmorph is an open-source software available in a git repository at
+https://github.com/diffpy/diffpy.pdfmorph.
+
+Feel free to fork the project and contribute! To install PDFmorph
+in a development mode where the source files are used directly
+rather than copied to a system directory, use ::
+
+     python setup.py develop --user
+
+
+CONTACTS
+------------------------------------------------------------------------
+
+For more information on PDFmorph, visit the project wep-page:
+
+# FIXME - make a page for the project
```

Comment:
@sbillinge @CJ-Wright @chiahaoliu do we have any idea when this page might be done by, or would it be ok to remove this for now? Also, would @chiahaoliu or @CJ-Wright be comfortable being added to this section as contacts?

Reply:
I am not sure how this is done, but maybe look around in diffpy repo in general.  I think we build the diffpy website from sources in there.  Use as a template maybe PDFgetX3 or PDFgui.

Reply:
on it - I'll see about making one!

Reply:
We have a centralized repo for `www.diffpy.org` where we provide an overview of our packages - We would need an introductory page there. Example of pdfgetx: https://github.com/diffpy/diffpy.github.io/blob/source/products/pdfgetx.rst

We would also need a more detailed doc in this `pdfmorph` repo.

Reply:
would it be worth putting up a skeleton for it at the moment and building onto it later? In order to do that, would I just add another rst file to the products folder of the repo you liked @chiahaoliu ?

Reply:
All we need to close this issue is to have the link to the page finalized (and correct).  We should make a new issue for writing that web-page (but that issue must be on the release milestone, so on the critical path for release). Something like:
``https://github.com/diffpy/diffpy.github.io/blob/source/products/pdfmorph.rst``

The actual link can (should?) be hidden under some more readable text.

Reply:
If we add the `pdfmorph.rst` file to the `diffpy.github.io` repo, will it automatically create that webpage? I don't want to mess anything up but if that's the case I'll start a new Issue and PR to get on that ASAP